### PR TITLE
Add description feature about acquisition place

### PR DESCRIPTION
**Pull request by sonohoshi**
### Description

SSIA

[caution] This PR has changes of #1294 

### How to test

1. Click any material item's icon.
2. Click the Question mark button.

### Related Links

[[UI] 획득경로 툴팁 도움말 팝업](https://app.asana.com/0/958521740385861/1202018099721425/f)

### Required Reviewers

@planetarium/9c-dev 

### Screenshot

image
image

### CODE REVIEW

File: nekoyume/Assets/_Scripts/UI/Module/AcquisitionPlaceDescription.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        public void Show(RectTransform parentTransform, RectTransform helpButtonTransform)
+        {
+            gameObject.SetActive(true);
+            var parentWorldPos = parentTransform.GetWorldPositionOfCenter();
+            var parentScreenPos = _mainCamera.WorldToScreenPoint(parentWorldPos);
+            if (parentScreenPos.x <= _mainCamera.pixelWidth / 2)
+            {
+                leftDeco.gameObject.SetActive(true);
+                rightDeco.gameObject.SetActive(false);
+                contentsRectTransform.MoveToRelatedPosition(helpButtonTransform, PivotPresetType.BottomRight, new float2(200f, 0f));
```
Comment (Unengine) [inline nekoyume/Assets/_Scripts/UI/Module/AcquisitionPlaceDescription.cs:40]: Is it ok to move with constant parameter?

Comment (sonohoshi) [reply nekoyume/Assets/_Scripts/UI/Module/AcquisitionPlaceDescription.cs:40]: OK. I'll reflect your sugestion. THX 👍

Comment (Unengine) [reply nekoyume/Assets/_Scripts/UI/Module/AcquisitionPlaceDescription.cs]: I mean, i think it should handle various screen resolution.

Comment (sonohoshi) [reply nekoyume/Assets/_Scripts/UI/Module/AcquisitionPlaceDescription.cs]: ah ok I understand... I'll look for another way. Thank you.

Comment (sonohoshi) [reply nekoyume/Assets/_Scripts/UI/Module/AcquisitionPlaceDescription.cs]: @Unengine I tested it. The result was good. I confirmed that there is no problem with various resolutions such as FHD, QHD, and 4K.

**Review by Unengine — APPROVED** [review]

**Pull request merged**